rh-tictactoe/src/app/screens/Login/styles.scss

```diff
@@ -0,0 +1,20 @@
+.login-form {
+  align-items: center;
+  border: 1px solid black;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 30px;
+  width: 300px;
+}
+
+.submit-button {
+  background-color:#008CBA; 
+  border-radius: 5px;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 16px;
```

Comment:
we can leave all size values here for now, but remember that the best way is creating a `dimensions` file inside constants with all sizes. We can talk about this later